samples/bluetooth/llpm/README.rst

```diff
@@ -0,0 +1,313 @@
+.. _ble_llpm:
+
+Bluetooth: LLPM
+###############
+
+The Bluetooth Low Latency Packet Mode (LLPM) sample uses the :ref:`latency_readme` to showcase the LLPM proprietary Bluetooth extension from Nordic Semiconductor.
```

Comment:
Would rephrase it to "The Bluetooth Low Latency Packet Mode (LLPM) sample uses the :ref:`throughput_readme` to measure the GATT Write latency. It uses the properietary LLPM Bluetooth extensions to achieve a latency below 7.5 ms. The sample can also be used to determine how different connection parameters affect the latency"

Reply:
Agree - I was looking for an information that this is not a Bluetooth standard.